{{ cookiecutter.repo_name }}/{{ cookiecutter.repo_name }}/settings/dev.py

```diff
@@ -9,19 +16,27 @@
     'DRTOOLS_SETTINGS_MODULE': '{{ cookiecutter.repo_name }}.settings.dev'
 }
 
-_modules_path = os.environ.get(
-    'MODULES_PATH',
-    os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
-)
-CONTAINER_TASK_VOLUMES = {
-    os.path.join(_modules_path, 'sparsity'):
-        {'bind': '/home/drtools/sparsity', 'mode': 'ro'},
-    os.path.join(_modules_path, 'drtools'):
-        {'bind': '/home/drtools/drtools', 'mode': 'ro'},
-    os.path.join(_modules_path, '{{ cookiecutter.repo_name }}'):
-        {'bind': '/home/drtools/{{ cookiecutter.repo_name }}', 'mode': 'ro'},
-}
+try:
+    _modules_path = Path(os.environ['MODULES_PATH'])
+
+    _sparsity = _modules_path.joinpath('sparsity')
+    _drtools = _modules_path.joinpath('drtools_base')
+
+    CONTAINER_TASK_VOLUMES.update({
+        str(_modules_path.joinpath('{{ cookiecutter.repo_name }}')):
+            {'bind': '/home/drtools/{{ cookiecutter.repo_name }}',
+             'mode': 'ro'},
+    })
```

Comment:
Could you please make these 3 bindings consistent? I mean, either use `.update` one time or `__setitem__` 3 times. And either create 3 helper variables (`_sparsity`, `_drtools` and e.g. `_project`) or don't create any.